### DISCUSSION TITLE
How to remove spaces to the right of row?

### DISCUSSION THREAD

**Discussion by dninomiya** [suren-atoyan/monaco-react#559] [Q&A]
I'm trying to implement an editor that makes the size of the editor fit the content.
There is a 41px margin on the right side of the row, so it cannot fit the content accurately.
Is there a way to remove the right margin?

thank you for your support!

![image](https://github.com/suren-atoyan/monaco-react/assets/5842851/d37a492e-416b-4609-a0b7-3fd173981623)

**Comment by suren-atoyan** [comment]
Check the API [here](https://microsoft.github.io/monaco-editor/typedoc/modules/editor.html), you may find something. I've checked a few things, but could find a solution 🤔
